frontend/src/App.js

```diff
@@ -12,6 +12,7 @@ import User from "./components/User/User";
 
 import "./App.scss";
 import "./config"
+import EditProfile from "./components/EditProfile/EditProfile";
 
 const setTokensAndLogin = (response) => {
```

Comment:
Just a question: is there a reason this isn't included in the App arrow function?

Reply:
Not sure, but I moved it inside. Doesn't seem to cause any issue